oelint_adv/__main__.py

```diff
@@ -140,6 +145,13 @@ def group_files(files):
 
     return res.values()
 
+def print_ruleset(args):
+    rules = [x for x in load_rules(args,
+        add_rules=args.addrules, add_dirs=args.customrules)]
+    ruleset = {r.ID: r.Severity for r in rules}
```

Comment:
As we have potentially multiple IDs per rules (and each of them can have a different severity, depending on how the __rulefile__ is configured) I would like to see that part here as a new function of `Rule` class... maybe something like `.get_active_ruleset_json()`

Reply:
Initially, I added all the IDs, but then I noticed they weren't picked up at all - only the "main ID" did anything... Maybe I missed something?

Reply:
Are you sure? For instance `oelint.var.mandatoryvar` rule should be translated to at least 4 IDs like `oelint.var.mandatoryvar.DESCRIPTION` a.s.o. - each can be controlled in severity by specifying it in the rules on its own

Reply:
Huch, that doesn't look right and is likely an unrelated bug - could you please file one for that? thx

Still I think it's best to return the result after the `appendix` explode

Reply:
See issue #236.

> Still I think it's best to return the result after the appendix explode

Okay, if the intended behavior was that those sub-rules are also configurable, then it makes sense.

Reply:
Bug is fixed - I will wait till tomorrow (18:00 UTC) before making a new release, just to give you a bit of time to get your changes in

Reply:
Found another issue with this, see #237.  The update to this PR here will now depend on the other one so you should probably merge the other one first (its commit shows up here, too).